build.gradle

```diff
@@ -28,13 +28,20 @@ repositories {
     }
 }
 
+configurations.all {
+    resolutionStrategy {
+        force "com.android.support:support-annotations:${supportLibraryVersion}"
```

Comment:
To fix this conflict which has been triggered while adding the Exoplayer library dependency:

Conflict with dependency 'com.android.support:support-annotations' in project 'android'. Resolved versions for app (25.0.1) and test app (24.2.1) differ.